### PR TITLE
Feature/sync committees info (BIDS-1153)

### PR DESCRIPTION
**Pull request by D13ce**

### CODE REVIEW

File: src/app/utils/ValidatorUtils.ts [hunk 1]
```diff
@@ -370,13 +364,28 @@ export class ValidatorUtils extends CacheModule {
...
 		const result = request.parse(response)[0]
-		this.lastFreshTime = Date.now()
 		this.currentEpoch = result.currentEpoch[0]
 		this.olderEpoch = result.olderEpoch[0]
 		this.rocketpoolStats = result.rocketpool_network_stats[0]
 		const validatorEffectivenessResponse = result.effectiveness
 		const validatorsResponse = result.validators
 
+		this.syncCommitteesStats = null
+		const hideSyncCommitteesStats = true // wait for the explorer to show sync committee stats too before showing them in the app
```
Comment (manuelsc) [inline src/app/utils/ValidatorUtils.ts:374]: I don't think this is necessary. We can draft the app release once it's live on the explorer

File: src/app/utils/ValidatorUtils.ts [hunk 2]
```diff
@@ -370,13 +364,28 @@ export class ValidatorUtils extends CacheModule {
...
+		if (result.sync_committees_stats && !hideSyncCommitteesStats) {
```
Comment (manuelsc) [inline src/app/utils/ValidatorUtils.ts:375]: Please move calculations to the OverviewController

**Review by manuelsc — CHANGES_REQUESTED** [review]

File: src/app/utils/ValidatorUtils.ts (hunk 1 above)
Comment (D13ce) [reply src/app/utils/ValidatorUtils.ts:374]: As discussed, we will not delay the sync committee luck on the app. I have removed the `hideSyncCommitteesStats` special handling.

File: src/app/utils/ValidatorUtils.ts (hunk 2 above)
Comment (D13ce) [reply src/app/utils/ValidatorUtils.ts:375]: Good suggestion, done.

**Review by manuelsc — DISMISSED** [review]
utACK

**Review by Nellyjay22 — DISMISSED** [review]

**Review by manuelsc — APPROVED** [review]
lgtm

**Comment by sarinmsari** [comment]
This PR has a lot of useful context. I tried structuring it into a short doc so it’s easier to revisit later:
[https://gist.github.com/doopstech/e9fd5479563b997a32abd34b0284a378](https://gist.github.com/doopstech/e9fd5479563b997a32abd34b0284a378)

If this isn’t appropriate here, feel free to ignore, just experimenting with better ways to preserve PR context.

**Pull request merged**